plugins/utils/virt-column.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  helpers,
+  pkgs,
+  config,
+  ...
+}:
+with lib;
+  helpers.neovim-plugin.mkNeovimPlugin config
+  {
+    name = "virt-column";
+    originalName = "virt-column.nvim";
+    defaultPackage = pkgs.vimPlugins.virt-column-nvim;
+
+    maintainers = [helpers.maintainers.alisonjenkins];
+
+    settingsOptions = {
+      enabled = helpers.defaultNullOpts.mkBool true "Enables or disables virt-column";
```

```suggestion
      enabled = helpers.defaultNullOpts.mkBool true ''
        Enables or disables virt-column.
      '';

```

Sorry to bother you with this, but it is better to follow some common styling across the code base:
- A blank line between each option
- Description on a separate line (using double single quotes `''`)
- Descriptions start with a capital letter and end up with a period.